### ISSUE TITLE
No Related Resource link between PVCs and Pods

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#5653]
Internal reference: SURE-4367
Reported in 2.5.12 & 2.6.4

Issue description:

The UI has a "Related Resources" tab on PVCs and Pods, but there is no related resource for pods that consume a PVC

Repro steps:

- Create Pod that consumes a PVC
- View the "Related Resources" tab on that PVC

Workaround:
Is workaround available and implemented? no

Actual behavior:

There is no reference to the Pod that consumes that PVC

Expected behavior:

The Pod would be considered a related resource, so should show up as one.

**Comment by nwmac** [comment]
Blocked on backend ticket: https://github.com/rancher/rancher/issues/37599

**Comment by catherineluse** [comment]
Do we need it to be in the related resources tab? We could create separate tabs for them, similar to Services and Ingresses on the workload detail page. We already have access to all the Pods, PVs and PVCs, and we know which Pods are bound to PVs. I think we already have enough info to add a Pods tab to the PVC detail page which could show which PVs the Pods are bound to.

**Comment by nwmac** [comment]
There should be no UI work - leaving this open to validate once the backend work is done

**Comment by hhstu** [comment]
Pod to pvc don`t have ownerReferences, not like pod  to deployment. anotherway to fix it  is list all pod of  namespace and  filter which pod is use pvc. I have fix it in my environment, although it is not elegant enough

![image](https://github.com/rancher/dashboard/assets/55697154/ffe34f5a-9321-4a44-8150-946edb39d6ad)

**Comment by nwmac** [comment]
Closing as won't fix
